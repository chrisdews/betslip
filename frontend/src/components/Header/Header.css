.header {
    background-color: aqua;
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 0em 1em 0em 1em;
}

