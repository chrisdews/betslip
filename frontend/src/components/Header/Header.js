import React from "react";
import PropTypes from "prop-types";
import Dropdown from "../Dropdown";
import "./Header.css";

function Header({ setOddsFilter }) {
  return (
    <div className="header">
      <h1>Betslip</h1>
      <Dropdown setOddsFilter={setOddsFilter} />
    </div>
  );
}

Header.propTypes = {};

export default Header;
