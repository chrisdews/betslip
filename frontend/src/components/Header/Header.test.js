import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';
import { render, screen } from '@testing-library/react'

xdescribe ('Header', ()=>{
    it('renders correctly - snapshot', () => {
        const tree = renderer
          .create(<Header><h1>Betslip</h1></Header>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });

    test('renders Betslip title', () => {
        render(<Header><h1>Betslip</h1></Header>)
        expect(screen.getByText('Betslip'))
    })
})