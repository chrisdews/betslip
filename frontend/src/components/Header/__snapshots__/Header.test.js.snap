// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly - snapshot 1`] = `
<div>
  <div
    class="sc-dlnjwi oaTBY"
  >
    <div
      class="sc-jSFjdj dqnCUv"
    >
      <h1
        class="sc-eCApnc bGVqgi"
      >
        BETSLIP
      </h1>
      <img
        class="sc-hKFxyN eaodwR"
        src="oc-logo.png"
      />
    </div>
    <select
      class="sc-bdnxRM einyrw"
      data-testid="dropdown-select"
      id="selectList"
      name="selectList"
    >
      <option
        class="sc-gtsrHT DFeHs"
        value="all"
      >
        ALL
      </option>
      <option
        class="sc-gtsrHT DFeHs"
        value="decimalOddsLessThanTwo"
      >
        Odds less than 2
      </option>
      <option
        class="sc-gtsrHT DFeHs"
        value="decimalOddsMoreThanTwo"
      >
        Odds greater than 2
      </option>
    </select>
  </div>
</div>
`;
